imdb_django/management/commands/import_akas.py

```diff
@@ -0,0 +1,84 @@
+import csv
+
+from django.core.management.base import BaseCommand
+from django.db import transaction
+from django.db.utils import IntegrityError  # Import IntegrityError
+
+from imdb_django.models import (
+    TitleAka,
+)  # Replace 'your_app_name' with your actual app name
+
+
+class Command(BaseCommand):
+    help = "Load data from TSV file into MySQL database"
+
+    def add_arguments(self, parser):
+        parser.add_argument("tsv_file", type=str, help="Path to the TSV file")
+
+    def handle(self, *args, **options):
+        tsv_file_path = options["tsv_file"]
+
+        with open(tsv_file_path, "r", encoding="utf-8") as tsv_file:
+            tsv_reader = csv.DictReader(tsv_file, delimiter="\t")
+
+            with transaction.atomic():  # Use a transaction for database consistency
+                row_count = 0  # Initialize a row count
+
+                for row in tsv_reader:
```

Comment:
Please have a look at this https://stackoverflow.com/a/522578

We might not need to increment `row_count` manually